agate/pattern-virtualgridlist-api/src/components/SideBar/SideBar.js

```diff
@@ -31,7 +31,7 @@ const SideBar = kind({
 		return (
 			<Group
 				{...rest}
-				childComponent={SelectableItem}
+				childComponent={RadioItem}
```

Comment:
```suggestion
				childComponent={RadioItem}
```
You can use Item or Button

Reply:
I changed RadioItem to Item.